app/src/main/kotlin/com/mapbox/vision/teaser/MainActivity.kt

```diff
@@ -209,7 +209,7 @@ class MainActivity : AppCompatActivity(), ReplayModeFragment.OnSelectModeItemLis
         }
     }
 
-    private fun tryToInitVisionManager() {
+    private fun startVision() {
```

Comment:
On the one hand I like it more (as it is shorter).
On the other hand it indeed could NOT start vision at all.
My opinion is to leave `startVision` but add some warning to logcat if it could not get started.

Reply:
Will add the logs, sure. 
start/stop is also more consistent besides of being shorter

Reply:
Checked it one more time, not sure we need any logs here. We expect that it can fail in certain conditions so I'd keep as is.